bugbot/rules/perfalert_inactive_regression_nag.py

```diff
@@ -0,0 +1,135 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from libmozdata.bugzilla import Bugzilla
+
+from bugbot import logger, utils
+from bugbot.rules.needinfo_regression_author import NeedinfoRegressionAuthor
+from bugbot.user_activity import UserActivity, UserStatus
+
+
+class PerfAlertInactiveRegressionNag(NeedinfoRegressionAuthor):
+    def __init__(self, nweeks=1):
+        super().__init__()
+        self.nweeks = nweeks
+
+    def description(self):
+        return f"PerfAlert regressions nag with {self.nweeks} week of inactivity"
+
+    def handle_bug(self, bug, data):
+        if len(bug["regressed_by"]) != 1:
+            # either we don't have access to the regressor,
+            # or there's more than one, either way leave things alone
+            return
+
+        bug_id = str(bug["id"])
+        data[bug_id] = {
+            "creator": bug["creator"],
+            "regressor_id": bug["regressed_by"][0],
+            "severity": bug["severity"],
+        }
+
+        if "triage_owner_detail" in bug:
+            data[bug_id]["triage_owner"] = bug["triage_owner_detail"]
```

Comment:
It does not seem that we are using the triage owner info in this rule.
```suggestion
```

Reply:
Ah ok, I was going to leave for when we come back to handle the triage owner needinfo - but we can just re-add it when the time comes.